packages/core/src/RangeCalendar/RangeCalendarViewTrigger.vue

```
@@ -0,0 +1,49 @@
+    headingValue: string
+    /** The active view */
+    view: CalendarUnit
+    /** Whether the trigger is disabled (at `maxView`, or the calendar is disabled) */
+    disabled: boolean
+  }) => any
+}
+</script>
+
+<script setup lang="ts">
+import { Primitive } from '@/Primitive'
+import { getCalendarViewTriggerSurface } from '@/Calendar/useCalendar'
```

ESLint

[error] Expected "`@/Calendar/useCalendar`" to come before "`@/Primitive`".

(perfectionist/sort-imports)